lib/matplotlib/backend_bases.py

```diff
@@ -600,26 +600,33 @@ def _draw_text_as_path(self, gc, x, y, s, prop, angle, ismath):
         gc.set_linewidth(0.0)
         self.draw_path(gc, path, transform, rgbFace=color)
 
+    def _get_text_width_height_descent_tex(self, s, prop):
```

Comment:
I guess you don't really need to split this out, just doing a super call (as previously done in the Cairo backend) is enough? The advantage is that this makes it clearer that this is not a method that third party backends typically need to override.

Reply:
Sure! I think the same can often be done for `ismath=True` as well.

Reply:
I would like to do the same thing for `ismath=True`, but the dpi aspect makes it not so trivial...